src/jokes/models/error.py

```diff
@@ -18,16 +17,7 @@ def debug(self) -> bool:
         the thread nor the value exist, returns
         False.
         """
-        return context.obj.get("DEBUG") if (context := self.get_context()) else False
-
-
-    def get_context(self) -> Context | None:
-        """
-        Gets the current thread's context. If no
-        context exists, silently fails and returns
-        None instead.
-        """
-        return get_current_context(silent=True)
+        return context.obj.get("DEBUG") if (context := get_current_context(silent=True)) else False
```

Comment:
This pattern of adding an object to the context and fetching it this way is repeated (and will be repeated again if the submit dry-run is ever fully implemented).

Maybe add a new util? `JokesContext`?

Reply:
Context can already be fetched from anywhere in the code via `get_current_context`.

We could add a method like this though:

```py
def get_context_object(key: str, default: Any) -> Any:
    return c.obj.get(key) if (c := get_current_context(silent=True)) else False
```